ngrok/src/tunnel_ext.rs

```diff
@@ -82,6 +90,29 @@ where
     Ok(())
 }
 
+async fn forward_unix_conns<T, F>(
+    this: &mut T,
+    addr: String,
+    mut on_err: F,
+) -> Result<(), io::Error>
+where
+    T: Tunnel + ?Sized,
+    F: FnMut(io::Error, Conn),
+{
+    let span = Span::current();
+    let path = Path::new(addr.as_str());
+    span.record("path", field::debug(&path));
+    trace!("looked up local path");
```

Comment:
Can probably ditch this message, since nothing actually happened. In the tcp case, there's a maybe-hostname that gets resolved to actual addrs.

Reply:
Removed.